### ISSUE TITLE
How can I use this app in english?

### ISSUE THREAD

**Issue by rohankhanna** [gragrance/CaptuocrToy#19]
Everything is in Mandarin. And Although I can understand what each item does through trial and error, I'd rather have an english option. I don't mind translating if someone were to point me in the right direction

**Comment by zeptolee** [comment]
are you still need english language support?

**Comment by bboomts** [comment]
Like @rohankhanna I discover how to use it but I would also love an English version for ease of use. I could also help with translation if I knew how I could do that.

**Comment by rohankhanna** [comment]
@zeptolee Yes I would, although My macbook is a few months away from me, so I wouldn't be able to test it soon enough.
